client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/RequestLineEdit.tsx

```diff
@@ -1,452 +1,123 @@
-import React from 'react';
+import React, { ReactElement } from 'react';
 import { useTranslation } from '@common/intl';
 import {
   ItemRowFragment,
-  ReasonOptionsSearchInput,
   StockItemSearchInput,
 } from '@openmsupply-client/system';
 import {
-  BarIcon,
   Box,
   FnUtils,
-  InputWithLabelRow,
   InsertRequestRequisitionLineInput,
-  NumericTextInput,
-  NumUtils,
-  Popover,
-  ReasonOptionNodeType,
-  Switch,
   TextArea,
-  useAuthContext,
-  useNavigate,
   usePluginProvider,
-  useTheme,
-  useToggle,
   useWindowDimensions,
 } from '@openmsupply-client/common';
 import { DraftRequestLine } from './hooks';
-import { Footer } from './Footer';
-import { RequestStats } from './ItemCharts/RequestStats';
 import { RequestFragment, RequestLineFragment } from '../../api';
-import { buildItemEditRoute } from '../utils';
-
-const INPUT_WIDTH = 100;
-const LABEL_WIDTH = '150px';
+import { AccordionPanelSection } from '@openmsupply-client/invoices/src/Prescriptions/LineEditView/PanelSection';
```

Comment:
Probs good to be done in a different issue, move `AccordionPanelSection` to `common` components directory

Reply:
definitely!